src/kitovu/utils.py

```diff
@@ -24,6 +24,26 @@ def get_password(plugin: str, identifier: str) -> str:
     return password
 
 
+def sanitize_filename(name: str, replacement: typing.Optional[str] = '_') -> str:
```

Comment:
Might be more in line with what we do elsewhere to take a `pathlib.PurePath` (and return one) here instead.